src/gui/dialogs/title_screen.hpp

```diff
@@ -61,6 +61,9 @@ class title_screen : public modal_dialog
 		QUIT_GAME,
 		// Used to reload all game data
 		RELOAD_GAME_DATA,
+		// Dummy retval used to reshow the titlescreen,
```

Comment:
Is it really a dummy value if it has a clear function?

Reply:
removed "Dummy retval".